problem/undefined-to-null_en.md

```diff
@@ -1,4 +1,145 @@
+## Problem 176. Undefined to null
```

Comment:
title is not necessary please remove it,
instead please credit yourself like this https://github.com/BFEdev/BFE.dev-solutions/blob/main/problem/create-your-own-promise_en.md